### PR TITLE
Resolve ordering problem because of Virtual Events

### PR DESCRIPTION
**Pull request by bordoni**
🎫 [ECP-598]
---
📹 https://i.bordoni.me/8LunO61R

[ECP-598]: https://theeventscalendar.atlassian.net/browse/ECP-598

### CODE REVIEW

File: src/Tribe/Repository.php [hunk 1]
```diff
@@ -752,7 +752,11 @@ public function order( $order = 'ASC' ) {
 	 */
 	public function order_by( $order_by, $order = 'DESC' ) {
 		$this->query_args['orderby'] = $order_by;
-		$this->query_args['order']   = $order;
+
+		// Based on `WP_Query->parse_orderby` we should ignore the global order passed, and use the value on for each item in array.
+		if ( ! is_array( $order_by ) ) {
+			$this->query_args['order']   = $order;
```
Comment (sc0ttkclark) [inline src/Tribe/Repository.php:758]:
```suggestion
			$this->query_args['order'] = $order;
```

**Review by sc0ttkclark — APPROVED** [review]
One whitespace tweak

**Review by lucatume — DISMISSED** [review]
Add tests coverage, then good to go.

File: tests/wpunit/Tribe/Repository/OrderTest.php [hunk 2]
```diff
@@ -43,7 +43,7 @@ public function should_allow_setting_the_order_direction_in_the_orderby_method()
 	}
 
 	/**
-	 * It should override the order direction specified w/ order w/ the orderby one
+	 * It should override the order direction specified w/ order w/ the orderby one.w
```
Comment (Camwyn) [inline tests/wpunit/Tribe/Repository/OrderTest.php:46]: w?

File: src/Tribe/Repository/Core_Read_Interface.php [hunk 3]
```diff
@@ -149,7 +149,7 @@ public function order( $order = 'ASC' );
 	 *
 	 * @since 4.7.19
 	 *
-	 * @param string $order_by The post field, custom field or alias key to order posts by.
+	 * @param string|string[] $order_by The post field, custom field or alias key to order posts by.
```
Comment (Camwyn) [inline src/Tribe/Repository/Core_Read_Interface.php:152]: `string|array<string,mixed>` (mixed or string?)

**Pull request merged**